authors/apps/articles/tests/test_report_article.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import status
+from authors.apps.articles.tests.base_class import ArticlesBaseTest
+from .test_data import valid_reported_article
+from authors.apps.articles.models import Article
+from django.urls import reverse
+from authors.apps.profiles.models import Profile
+
+class TestReportArticle(ArticlesBaseTest):
+    def test_can_not_report_own_article(self):
+        """
+        Tests method tests a user who is authenticated can report his/her own article
+        """
+
+        self.add_article()
+
+        article = Article.objects.all().first()
+        response = self.client.post(reverse('articles:report-article',
+                                           kwargs={'slug': article.slug}),
+                                    data=valid_reported_article,
+                                    format='json')
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertEqual(response.data['message'],'You cannot report your own article')
+    def test_can_report_some_article(self):
+        """
+        Tests method tests a user who is authenticated can report some article
+        """
+
+        self.add_article()
+        self.register_and_login_new_user()
+
+        article = Article.objects.all().first()
+        response = self.client.post(reverse('articles:report-article',
+                                           kwargs={'slug': article.slug}),
+                                    data=valid_reported_article,
```

Comment:
Please consider reviewing the response for easy reading i suggest kwargs should start at the same level as data

Reply:
I have worked to reformat that, however, the reason why kwargs is on a new line is so as to maintain lines less than 79 characters and since kwargs is still an argument of reverse, it still has to appear like its an argument of reverse for reverse to work on multiple lines.

If we examine where data is, it's essentially saying that even if it's on the third line it has to appear as an argument of post. I guess this is all because of the python way of indenting.